prisma/schema.prisma

```diff
@@ -34,13 +35,28 @@ model Password {
 }
 
 model Group {
-  id          String  @id @default(uuid())
+  id          String             @id @default(uuid())
   name        String
-  description String  @db.Text
+  description String             @db.Text
   imgUrl      String?
   imgAlt      String?
   events      Event[]
   users       User[]
+  socalLinks  GroupSocialLink[]
+}
+
+model GroupSocialLink {
+  groupId String @id @unique
+  group   Group  @relation(fields: [groupId], references: [id], onDelete: Cascade)
+  type    String
+  url     String
+}
+
+model UserSocialLink {
+  userId String @id @unique
+  User   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
```

Comment:
`onDelete: Cascade` is pretty cool, it deletes this entry, if the reference gets deleted

here are some other possible actions
![image](https://github.com/social-plan-it/plan-it-social-web/assets/29219684/9af0fd1e-3ece-4f78-968e-6190403ec80e)